src/api/controllers/misc.ts

```diff
@@ -27,11 +27,31 @@ export async function handleInternalReady(
   const payload = data.d as ReadyPayload;
   setBotID(payload.user.id);
 
-  // Triggered on each shard
+  cache.shardedGuilds += payload.guilds.length;
   eventHandlers.shardReady?.(shardID);
+
+  //Ability to fine tune
+  const chunk = 1200;
+  const buffer = 400;
+  const request = 7;
+
+  //Remaining guilds to load via handleInternalGuildCreate
+  function checkRemaining(left: any) {
```

Comment:
Try finding a corresponding type and not any,

Reply:
`number`